packages/FormElement/stories/FormElement.examples.stories.js

```diff
@@ -102,6 +102,13 @@ nestedExample.story = {
   parameters: exampleStoryParameters,
 };
 
+export const inlineContentExample = () => (
+  <FormElementStory storyName="Inline Content" fileName="examples/InlineContent.js">
```

Comment:
It will need the `component="FormElement"` prop for the `source` link to appear (it needs it to reconstruct the GitHub path/URL).